### PR TITLE
Set "proxy base url" for GeoServer

### PR DESCRIPTION
**Pull request by JakobMiksch**
- set the GeoServer proxy base url via the REST-API to `https://sauber-sdi.meggsimum.de/geoserver`
- this fixes the issue that the productive GeoServer points to internal Docker services instead of the correct URL

### CODE REVIEW

File: geoserver_init/index.js [hunk 1]
```diff
@@ -54,11 +55,31 @@ async function initGeoserver() {
 
   await createWorkspaces();
 
+  await setProxyBaseUrl();
```
Comment (chrismayer) [inline geoserver_init/index.js:58]: Please add this step to the list in the function API docs above (around line 49).

**Review by chrismayer — APPROVED** [review]
LGTM, one mini remark you might want to address. Thanks!

File: geoserver_init/index.js (hunk 1 above)
Comment (JakobMiksch) [reply geoserver_init/index.js:60]: done with f86b1bedff1fb5c2ceabdbd1e491ed659bb00e7b

**Comment by JakobMiksch** [comment]
@chrismayer thanks for  your review. I addressed your remark.

I additionally  added the "proxy base url"  to the logged variables with 1d8443d6f63fd192dd62c8e7d2795914876c3605

**Review by jdk24 — APPROVED** [review]
Thank you!! Please merge at will.

**Pull request merged**